external/implementations/samlconf-openam-impl/README.md

```diff
@@ -0,0 +1,58 @@
+| Does not return a SAML Response Error when given a Non-Matching destination | Core 3.2.1 | If [Destination] is present, the actual recipient MUST check that the URI reference identifies the location at which the message was received. If it does not, the request MUST be discarded.
+
+## Steps to Test OpenAM's IDP
+#### install and run OpenAM
+* Download [Apache Tomcat](https://tomcat.apache.org/download-70.cgi).
+* Download the OpenAM .war files from [OpenAM Releases](https://github.com/OpenIdentityPlatform/OpenAM/releases).
+* Rename OpenAM-\<version\>.war to OpenAM.war.
+* Unzip Apache Tomcat and move all OpenAm .war files into <tomcat>/webapps.
+* Run the following commands to start tomcat:
+  * `chmod -R 755 <tomcat>/bin`
+  * `export CATALINA_HOME=<tomcat>`
+  * `sh <tomcat>/bin/startup.sh`
+
+* Follow the steps [here](https://github.com/OpenIdentityPlatform/OpenAM/wiki/Quick-Start-Guide#basic-openam-setup) until Policy Configuration.
+
+#### Setup OpenAM as a hosted IDP
+* Once you've logged in as amAdmin, click Top Level Realm \> Configure SAMLv2 Provider \> Create Hosted Identify Provider.
+
+* Fill out the form (shown below) by picking a signing key, entering a name for the Circle of Trust (CoT) and clicking Configure.
+![Create A Hosted IdP](images/OpenAM-CreateIdP.png "Setting up an IdP")
+  * **NOTE**: "http://localhost:8080/OpenAM-14.1.13" would be the **baseURL** and "/" would be the **realmName** in the above example.
+
+#### Registering and Configuring the CTK
+* Register a new Service Provider under the same Circle of Trust (CoT) and give it the following metadata: [`samlconf-sp-metadata.xml`](../../../deployment/distribution/src/main/resources/samlconf-sp-metadata.xml)
+
+* Navigate to the SAML2 page on OpenAM, (Realm \> Applications \> SAML 2.0).
+* Click on the CoT that both the CTK and the IDP are a part of.
+* On the left under "available" there should be `https://samlhostdsa:8994`. Add it to the CoT and save.
+
+* Setup a new user for the SamlCTK to connect as, (Realm \> Subjects \> New...).
+  * ID: `ctk`
+  * Password: `gotestit`
+
+* If you're using the root realm ("/") you can skip the next two steps.
+  * Add the IDP metadata to the SamlCTK by going to `<baseUrl>/saml2/jsp/exportmetadata.jsp?realm=<realmName>`.
+  * Copy that XML into the [openam-idp-metadata.xml](src/main/resources/openam-idp-metadata.xml) file, replacing the pre-existing contents.
+
+* Configure the CTK to your OpenAM setup by filling out and modifying the following XML: [OpenAMLoginData.xml](src/main/resources/OpenAMLoginData.xml).
+
+* You're all set, (re)build the distribution and run the `samlconf` script under `deployment/distribution/build/install/samlconf/bin` with `-i ../implementations/openam -l -u ctk:gotestit`.
```

Comment:
✏️ Cut out the build part and start with "Run the `samlconf` script". These instructions are meant to be more geared towards just a user of the CTK.

Reply:
ahhh okay, I'll have to change up some of the paths then so they're editing the correct files.

Reply:
Yeah I hadn't thought about the fact that these paths were based around the source repo 😅

Reply:
Because of how the [OpenAMDataParser](https://github.com/codice/saml-conformance/pull/208/files#diff-a41bb74d5fdc5415b0102c03b92ed194R55), it loads the configuration xml from the jar file which needs to be rebuilt to reflect any changes made to it. The alternative would be to create an absolute path from a system property in the startup script (`samlconf`) to find where the xml config file lives so the user would never have to rebuild the SamlCTK. 
❓what are everyones thoughts on this matter?